packages/components/select/src/select.stories.ts

```diff
@@ -88,6 +198,9 @@ export const InForm: StoryObj = {
       sl-button-bar {
         grid-column: 1 / 3;
       }
+      sl-avatar {
+        margin: 0 4px;
+      }
     </style>
```

Comment:
Maybe we can change layout of the example a little bit to have the error message under the select element?
![image](https://github.com/sl-design-system/components/assets/111562742/492ec881-2402-420e-a018-9ff8a50c0929)
And also labels above select elements to have it consistent with other form examples? :)